examples/01-Modeling-Setup/Configurations.py

```diff
@@ -0,0 +1,127 @@
+"""
+General: configuration files
+----------------------------
+This example shows how you can use PyAEDT to export configuration files and re-use
+them to import in a new project. A configuration file is supported by these applications:
+
+* HFSS
+* 2D Extractor and Q3D Extractor
+* Maxwell
+* Icepak (in AEDT)
+* Mechanical (in AEDT)
+
+The following sections are covered:
+
+* Variables
+* Mesh operations (except Icepak)
+* Setup and optimetrics
+* Material properties
+* Object properties
+* Boundaries and excitations
+
+When a boundary is attached to a face, the tool tries to match it with a
+``FaceByPosition`` on the same object name on the target design. If, for
```

Comment:
What is ``FaceByPosition" as we should follow the code entity with a noun--such as attribute, property, object, or whatever.